src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -121,24 +127,46 @@ private String getHIUId(ConsentDetail consentDetail) {
         return signedObject.getSignature();
     }
 
-    private ConsentArtefact mapToConsentArtefact(ConsentRequestDetail consentRequest, GrantedConsent consent) {
+    private ConsentArtefact mapToConsentArtefact(ConsentRequestDetail requestDetail, GrantedConsent granted) {
+        PatientLinkedContext linkedPatientInfo = PatientLinkedContext.builder()
```

Comment:
Everywhere we are dealing with Patient only right, Having PatientLinkedContext is kind of redundant IMO.

Reply:
Done. Split with PatientReference and GrantedContext